lib/rules/shorthand-property-no-redundant-values/index.mjs

```diff
@@ -39,13 +40,14 @@ const SUPPORTED_SHORTHANDS = new Set([
 	'inset', 'inset-block', 'inset-inline',
 ]);
 
-/**
- * @param {string} property
- * @returns {boolean}
- */
-function isSupportedShorthand(property) {
-	return SUPPORTED_SHORTHANDS.has(property);
-}
+// prettier-ignore
+const FOUR_DIRECTIONAL_SHORTHAND_PROPERTIES = new Set([
```

Comment:
```suggestion
// prettier-ignore
/** @type {ReadonlySet<string>} */
const FOUR_DIRECTIONAL_SHORTHAND_PROPERTIES = new Set([
```

see stylelint/stylelint#8515 and prettier/prettier#5748

Reply:
Thank you, I've fixed it.
[refactor: add type tag](https://github.com/stylelint/stylelint/pull/8527/commits/a397f85ee668d0bbd1694a2043e97ede9a58452d)